public/docs/_examples/quickstart/ts/app/app.module.ts

```diff
@@ -1,13 +1,16 @@
 // #docregion
-import { NgModule }      from '@angular/core';
-import { BrowserModule } from '@angular/platform-browser';
+import { Component, NgModule } from '@angular/core';
+import { BrowserModule }       from '@angular/platform-browser';
 
-import { AppComponent }   from './app.component';
+@Component({
+  template: `<h1>Hello Angular!</h1>`,
+  selector: 'my-app'
+})
+export class AppComponent { }
```

Comment:
It is extra, and if you truly want minimal, then I think removing the `export` is smart. Otherwise, the the educated eye it will make them look for where it is used. This is not a tool to build on, right? If it was, we would make multiple files in the first place. I think, rather, that we want a simple starting point to explain A2 at the most simplest level. In that case, I think removing export works. Thoughts?